src/main/java/com/rackspace/salus/monitor_management/services/MonitorManagement.java

```diff
@@ -653,6 +653,10 @@ public void handleEnvoyResourceChangedInZone(@Nullable String tenantId,
     else {
       log.debug("No bound monitors were previously assigned to envoy={}", fromEnvoyId);
     }
+
+    // Also check to make sure there are no unassigned bound monitors
+    // This can happen if new monitors are created while all monitors in a zone are down
```

Comment:
is this the wording meant:
```suggestion
    // This can happen if new monitors are created while all poller-envoys in a zone are down
```